internal/cli/util/util.go

```diff
@@ -856,3 +856,14 @@ func BuildPodAntiAffinity(podAntiAffinityStrategy string, topologyKeys []string)
 
 	return podAntiAffinity
 }
+
+// AddDirToPath add a dir to the PATH environment variable
+func AddDirToPath(dir string) error {
+	p := os.Getenv("PATH")
+	if p == "" {
+		p = dir
+	} else {
+		p = dir + ":" + p
+	}
+	return os.Setenv("PATH", p)
+}
```

Comment:
should trim the `p`

Reply:
I have change the func to avoid that putting empty and `p` having space.